test/e2e/v3/plugin_cluster_test.go

```diff
@@ -131,45 +99,15 @@ var _ = Describe("kubebuilder", func() {
 				GenerateV3(kbc, "v1")
 				Run(kbc)
 			})
-			It("should generate a runnable project with v1beta1 CRDs and Webhooks", func() {
-				// Skip if cluster version < 1.15, when `.spec.preserveUnknownFields` was not a v1beta1 CRD field.
-				// Skip if cluster version >= 1.22 because pre v1 CRDs and webhooks no longer exist.
-				if srvVer := kbc.K8sVersion.ServerVersion; srvVer.GetMajorInt() <= 1 && srvVer.GetMinorInt() < 15 ||
-					srvVer.GetMajorInt() <= 1 && srvVer.GetMinorInt() >= 22 {
-					Skip(fmt.Sprintf("cluster version %s does not support project defaults ",
-						srvVer.GitVersion))
-				}
-
-				GenerateV3(kbc, "v1beta1")
-				Run(kbc)
-			})
 
 			It("should generate a runnable project go/v3 with v1 CRDs and Webhooks with restricted pods", func() {
-				// Skip if cluster version < 1.16, when v1 CRDs and webhooks did not exist.
-				// Skip if cluster version < 1.19, because securityContext.seccompProfile only works from 1.19
-				// Otherwise, unknown field "seccompProfile" in io.k8s.api.core.v1.PodSecurityContext will be faced
-				if srvVer := kbc.K8sVersion.ServerVersion; srvVer.GetMajorInt() <= 1 && srvVer.GetMinorInt() < 19 {
-					Skip(fmt.Sprintf("cluster version %s does not support v1 CRDs or webhooks"+
-						"and securityContext.seccompProfile", srvVer.GitVersion))
-				}
-
 				kbc.IsRestricted = true
 				GenerateV3(kbc, "v1")
 				Run(kbc)
 			})
 			It("should generate a runnable project with v1beta1 CRDs and Webhooks with restricted pods", func() {
-				// Skip if cluster version < 1.15, when `.spec.preserveUnknownFields` was not a v1beta1 CRD field.
-				// Skip if cluster version < 1.19, because securityContext.seccompProfile only works from 1.19
-				// Otherwise, unknown field "seccompProfile" in io.k8s.api.core.v1.PodSecurityContext will be faced
-				// Skip if cluster version >= 1.22 because pre v1 CRDs and webhooks no longer exist.
-				if srvVer := kbc.K8sVersion.ServerVersion; srvVer.GetMajorInt() <= 1 && srvVer.GetMinorInt() < 19 ||
-					srvVer.GetMajorInt() <= 1 && srvVer.GetMinorInt() >= 22 {
-					Skip(fmt.Sprintf("cluster version %s does not support project defaults "+
-						"and securityContext.seccompProfile", srvVer.GitVersion))
-				}
-
 				kbc.IsRestricted = true
-				GenerateV3(kbc, "v1beta1")
+				// GenerateV3(kbc, "v1beta1")
 				Run(kbc)
 			})
```

Comment:
We can remove the whole one since it is to do the same done above but with the CRD/Webhooks using the API v1beta1 which is no longer provided since k8s 1.22

Reply:
Just removed the whole `it(...)` test case if I'm understanding correctly.

Reply:
Yes, we can have the whole it(...)